Sources/TABResourceLoader/Operations/BaseAsynchronousOperation.swift

```diff
@@ -62,12 +63,14 @@ open class BaseAsynchronousOperation: Operation {
     execute()
   }
 
+  /// Method to be overriden with the implemenation of the work done on the operation
   open func execute() {
     assertionFailure("execute must overriden")
     finish()
   }
 
-  public final func finish(_ errors: [Error] = []) {
+  /// Call this method after all the work is done signal the completion of this operation
```

Comment:
Sorry yeah that makes sense now. Such a little word makes all the difference, ha! Looks good now.